docs/examples/two-layer.rst

```diff
@@ -3,7 +3,7 @@ Two Layer QG Model Example
 ==========================
 
 Here is a quick overview of how to use the two-layer model. See the
-:py:class:`pyqg.QGModel` api documentation for further details.
+:py:class:``pyqg.QGModel`` api documentation for further details.
```

Comment:
This points to a larger issue in the documentation workflow. In sphinx, ```  py:class:`pyqg.QGModel` ``` has a special meaning which links to the relevant part of the API documentation. But if you write this in a notebook and use ipynb, the quotes get escaped to ```  py:class:``pyqg.QGModel`` ``, which breaks the links. The way I fixed this last time was to manually edit the .rst file, but that is not a long-term solution.